utils/schemaversion/schemaversion.go

```diff
@@ -20,3 +21,12 @@ func (v SchemaVersion) Validate() error {
 	}
 	return nil
 }
+
+// GetSemverSchemaVersion returns a SchemaVersion from the provided string,
+// trimming the non-semver prefix according to schema versioning in the
+// cnabio/cnab-spec repo
+func GetSemverSchemaVersion(schemaVersion string) SchemaVersion {
+	r := regexp.MustCompile("^cnab-[a-z]+-(.*)")
+	match := r.FindStringSubmatch(schemaVersion)
+	return SchemaVersion(match[1])
```

Comment:
`match` can be an empty/nil slice if `schemaVersion` doesn't match the regexp, so I guess this function should return an error instead?

Reply:
Ok just found that it's used to initialize a var:
https://github.com/cnabio/cnab-go/pull/188/files#diff-36a2eeedaad2da2e03f68eb1036b88c2R23
Maybe could we move it to an `init()` function and handle the error here (even if I don't like `init` 😅 )?
WDYT?

Reply:
I refactored to add `GetDefaultSchemaVersion()` methods for both Bundles and Claims to replace the global variable approach.  WDYT?